perception/traffic_light_classifier/src/cnn_classifier.cpp

```diff
@@ -163,6 +163,10 @@ void CNNClassifier::postProcess(
       boost::algorithm::split(color_and_shape, label, boost::is_any_of("-"));
       element.color = label2state_[color_and_shape.at(0)];
       element.shape = label2state_[color_and_shape.at(1)];
+    } else if (label.find("crosswalk") != std::string::npos) {
+      // found "crosswalk" in label string
+      element.color = label2state_[label.substr(10)];
+      element.shape = tier4_perception_msgs::msg::TrafficLightElement::CIRCLE;
```

Comment:
It it required? There is no word `crosswalk` in lamp_labels_ped.txt.
```
❯ cat lamp_labels_ped.txt 
red
green
unknown⏎  
```

Reply:
Yes, it is not required. I will delete it.